meinberlin/apps/dashboard/views.py

```diff
@@ -204,7 +204,7 @@ class ModuleDeleteView(PermissionRequiredMixin, generic.DeleteView):
     model = module_models.Module
     success_message = _("The module has been deleted")
 
-    def delete(self, request, *args, **kwargs):
+    def form_valid(self, request, *args, **kwargs):
```

Comment:
does the call to `super().delete` also need to be changed to `super().form_valid` ?